modules/cudaarithm/src/precomp.hpp

```diff
@@ -60,4 +60,24 @@
 #  include <cufft.h>
 #endif
 
+#include <cuda_runtime.h>
+
+namespace cv { namespace cuda {
```

Comment:
It is better to place this code into lut.hpp (private header in "cudaarithm/src" folder).
To avoid storing of implementation details in `precomp.hpp` - lets keep here only common headers.